server/utils/licenceStatus.js

```diff
@@ -182,7 +182,8 @@ function getCaStageState(licence) {
       bassAccepted,
       bassWithdrawn,
       bassWithdrawalReason,
-      caRefused: approvalRelease.decision === 'Yes' && finalChecksRefusal.decision === 'Yes',
+      caRefused: finalChecksRefusal.decision === 'Yes',
+      refusalReason,
```

Comment:
this is being declared in `caStageState`  - refusalReason will be overridden in post approval stages to be DM reason or if not exisitng CA reason. There is a test that covers this.